### DISCUSSION TITLE
How to keep theme updated

### DISCUSSION THREAD

**Discussion by PanZiwei** [wangchucheng/hugo-eureka#59] [Q&A]
Hi, I got the error `error: pathspec 'eureka_v0.5.0' did not match any file(s) known to git` when I tried to keep the theme updated by using the command below:
```
cd themes/eureka
git checkout eureka_v0.5.0
git pull
```
Any idea on this update issue?
Thank you so much for your help!

**Comment by wangchucheng** [comment]
I think it is a misuse in git checkout. You can look up for how git checkout works. And it may solve this problem.


________________________________
发件人: PanZiwei <notifications@github.com>
发送时间: Wednesday, February 3, 2021 1:31:25 PM
收件人: wangchucheng/hugo-eureka <hugo-eureka@noreply.github.com>
抄送: Subscribed <subscribed@noreply.github.com>
主题: [wangchucheng/hugo-eureka] How to keep theme updated (#59)


Hi, I got the error error: pathspec 'eureka_v0.5.0' did not match any file(s) known to git when I tried to keep the theme updated by using the command below:

cd themes/eureka
git checkout eureka_v0.5.0
git pull


Any idea on this update issue?
Thank you so much for your help!

―
